packages/core/src/orders/client/postOrderItemActivities.js

```diff
@@ -0,0 +1,43 @@
+import client, { adaptError } from '../../helpers/client';
+import join from 'proper-url-join';
+
+/**
+ * @typedef {object} PostOrderItemActivityData
+ *
+ * @alias PostOrderItemActivityData
+ * @memberof module:orders/client
+ *
+ * @property {string} type - Type of activity.
```

Comment:
Would be nice if you could specify which types of activity is possible to choose from.